### ISSUE TITLE
Request: Easier/Quicker Audio Offset Function

### ISSUE THREAD

**Issue by robnz09** [boxeehacks/boxeehack#358]
Again, thanks heaps for all your awesome work on this hack.

I wondered whether it was possible to create a shortcut to adjust the audio sync. I am finding more and more frequently that I have sync issues with my files and going into the settings each time playing with the offset is distracting when you are trying to watch the start of a movie. Would there be a way to create a shortcut function ie. up & down on the boxee remote? or even just a change the offset slider to something that sits discreetly in the bottom corner when you open it (as opposed to sitting over the centre of the content).

Thanks!

**Comment by boxeehacks** [comment]
I was thinking of connecting this to some keys on the keyboard, similar to how VLC does it...

**Comment by robnz09** [comment]
Yeah, that would be awesome and would definitely make it a heap easier. Thanks heaps!
